runner/config_schema.yaml

```diff
@@ -1,54 +1,126 @@
 $schema: http://json-schema.org/schema#
 $id: http://illixr.github.io/schemas/config.json
-type: object
 definitions:
-  key_vals: &key_vals
-    default: {}
-    type: object
-    additionalProperties:
-      type: string    
+  path:
+    oneOf:
+      - type: string
+      - type: object
+        additionalProperties: false
+        required:
```

Comment:
We should add `name` here as well. It won't affect functionality but it will help readability. The git urls are not the most readable. The worst one is Hologram being HOTLab (although I agree that that's a one off).